climada/entity/exposures/base.py

```diff
@@ -546,17 +543,19 @@ def plot_scatter(self, mask=None, ignore_zero=False, pop_name=True,
         adapt_fontsize : bool, optional
             If set to true, the size of the fonts will be adapted to the size of the figure.
             Otherwise the default matplotlib font size is used. Default is True.
+        title : str, optional
+            a title for the plot. If not set `self.description` is used.
         kwargs : optional
             arguments for scatter matplotlib function, e.g.
-            cmap='Greys'. Default: 'Wistia'
+            cmap='Greys'
 
         Returns
         -------
         cartopy.mpl.geoaxes.GeoAxesSubplot
         """
         crs_epsg, _ = u_plot.get_transformation(self.crs)
-        title = "\n".join(self.tag.description)
-        cbar_label = f'Value ({self.value_unit})'
+        title = (self.description.strip() if isinstance(self.description, str) else ""
+                 ) if title is None else title
```

Comment:
`title = something if title is None else title` seems redundant to me. Also: What other type than `str` would `description` support? Should we maybe cast it to `str` in the constructor to avoid testing its type here?
```suggestion
        if title is None:
            title = self.description.strip()
```

Reply:
`None` is the default. So if you call it with a `title`-argument - fine, up to you, even if it's an empty string.
If you don't bother - also fine, then the title from the description is plotted.

Reply:
your suggestion doesn't work if the `description` is `None`. You would need another `if` block. From there on it's a question of taste I guess.

Reply:
Then let's keep it as simple as can be: Users need to take care of formatting their descriptions. So I suggest we just do:
```suggestion
    if title is None:
        title = self.description
```
This should also work when `description` is `None`, `ax.set_title(None)` should have no (negative) effect.

Reply:
some util.plot function doesn't like `None`. Hence I did `tjitle = self.description or None`.